docs/index.md

```diff
@@ -82,19 +82,16 @@ This allows you to rely on UI5 Tooling for UI5-specific build functionality and
 All available APIs are documented in the [UI5 Tooling API Reference](https://sap.github.io/ui5-tooling/api/index.html).
 
 ```js linenums="1"
-const {normalizer} = require("@ui5/project");
-const {builder} = require("@ui5/builder");
+import {graphFromPackageDependencies} from "@ui5/project/graph";
```

Comment:
Not sure whether we should also address the docs within this PR. I think this should be handled separately, as we currently don't create any docs from this branch and we still need to figure out how we want to provide docs for multiple versions. Maybe you can just move those changes into another PR.

Reply:
Thats true. I've removed the documentation part and created another branch https://github.com/SAP/ui5-tooling/tree/docs-esm which we can use in future